tests/raftstore/cluster.rs

```diff
@@ -357,7 +357,7 @@ impl<T: Simulator> Cluster<T> {
                     count += 1;
                     continue;
                 }
-                let l = self.query_leader(*store_id, region_id, Duration::from_millis(10));
```

Comment:
10ms is too small for a very slow environment.

Reply:
I remembered that @overvenus enlarges it before, but @BusyJay thought it was not fine.

Reply:
I suggest using a litter larger value, like 50ms, 1s seems too large here.

Reply:
Why do you think 1s is too large? What's the problem to let it pass in 1s?

Reply:
here if we can't query the leader, we may wait for 30s, a little long. and we the leader is lost, we may wait for 1s for another try, it may also be long.

Reply:
maybe wait 20ms, and retry 500 count?